product_docs/docs/edbcloud/beta/migration/index.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Migrating
+---
+
+EDB provides migration tools to bring data into EDB Cloud such as Migration Portal and Migration Toolkit. 
```

Comment:
There are a lot of things we should do to make this better over time.  However, since we do have some documentation from Gabriele for a dump/restore method of importing a Postgres database to EDB Cloud, it probably does make sense to at least mention the different source databases that can be migrated to EDB Cloud and then either as note or a separate section describe the options for migrating EPAS and PG, with the dump/restore method being one of them.  

It is also possible to migrate an EPAS or PG database to EDB Cloud using MTK, Replication Server, or other methods.  The main difference with EPAS/PG migrations to EDB Cloud is that no schema conversions should be needed.  Therefore, Migration Portal wouldn't be used.  Currently, Migration Portal is only relevant for Oracle to EPAS migrations, where the target EPAS database could be deployed on-premises, in self-managed IaaS environment, or EDB Cloud.

In addition to Oracle, EPAS, and PostgreSQL as possible migration sources, there are some options sans database code conversion for migrating SQL Server and MySQL databases as well.  Would these need to be mentioned here as well?

Reply:
I'm ambivalent on the SQL Server and MySQL bits, but I think we need to structure the doc to get at the bits you're describing here.

Reply:
I took a crack at this.